pixi.toml

```diff
@@ -11,10 +11,11 @@ platforms = ["win-64", "linux-64"]
 
 [tasks]
 # Installation
-install-julia = "juliaup add 1.10.5 && juliaup override unset && juliaup override set 1.10.5"
+install-julia = "juliaup add 1.10.7 && juliaup override set 1.10.7"
 # Julia
-update-registry-julia = "julia --eval='using Pkg; Registry.update()'"
-instantiate-julia = {cmd = "julia --project --eval='using Pkg; Pkg.instantiate()'",env = { JULIA_SSL_CA_ROOTS_PATH = "" } }
+# Clear SSL_CERT_DIR to avoid Julia warnings, see https://github.com/JuliaLang/Downloads.jl/issues/244
+update-registry-julia = { cmd = "julia --eval='using Pkg; Registry.update()'", env = { SSL_CERT_DIR = "" } }
+instantiate-julia = {cmd = "julia --project --eval='using Pkg; Pkg.instantiate()'",env = { SSL_CERT_DIR = "" } }
```

Comment:
These will be less noisy, see https://github.com/JuliaLang/Downloads.jl/issues/244